onnx/checker.cc

```diff
@@ -216,25 +216,51 @@ void check_graph(const GraphProto& graph, int ir_version) {
   for (const auto& value_info : graph.output()) {
     check_value_info(value_info, ir_version);
   }
-  for (const auto& node : graph.node()) {
-    try {
-      check_node(node, ir_version);
-    } catch (ValidationError& ex) {
-      ex.AppendContext("Bad node spec: " + node.ShortDebugString());
-      throw ex;
-    }
-   }
 
-  std::unordered_set<std::string> input_names{};
+  std::unordered_set<std::string> output_names{};
   for (const auto& value_info : graph.input()) {
-    input_names.insert(value_info.name());
+    output_names.insert(value_info.name());
```

Comment:
want to check no duplicate graph input name please?

Reply:
Sorry no, because it's not the intention of this PR. Feel free to do it in a separate one.